extra/1-radio-stations.js

```diff
@@ -25,7 +31,11 @@
  * - Return only the frequencies that are radio stations.
  */
 // `getStations` goes here
-
+function getStations() {
+  getAllFrequencies().forEach((frequency) => {
+    isRadioStation(frequency);
+  });
```

Comment:
you can use the filter method for this function to help it pass the test. array.filter(functionName). It will look through the values within the array using the helper function, find the ones that are true, and return the value of the true elements within the array. 
https://www.w3schools.com/jsref/jsref_filter.asp

Reply:
it works thanks for your solution.